apps/meteor/app/livechat/server/api/lib/departments.ts

```diff
@@ -169,19 +169,18 @@ export async function findDepartmentAgents({
 	departmentId,
 	pagination: { offset, count, sort },
 }: FindDepartmentAgentsParams): Promise<PaginatedResult<{ agents: ILivechatDepartmentAgents[] }>> {
-	const { cursor, totalCount } = LivechatDepartmentAgents.findAgentsByDepartmentId<ILivechatDepartmentAgents>(departmentId, {
```

Comment:
Would u mind checking if after using this new function the old one is still used somewhere? if not, think we can remove it